src/components/Courses.js

```diff
@@ -1,87 +1,158 @@
 import React, { Component } from 'react'
-import Card from './Card.js'
-
-
-let courseData = [
-    {
-        "id": 1,
-        "image": "https://img-c.udemycdn.com/course/240x135/394676_ce3d_5.jpg",
-        "title": "Learn Python: The Complete Python Programming Course",
-        "instructor": "Avinash Jain,The Codex",
-        "rating": ["4.4", "2,845"],
-        "sale": true,
-        "original-price": "679.99",
-        "new-price": "199.99",
-        "best-seller": false
-    },
-    {
-        "id": 2,
-        "image": "https://img-c.udemycdn.com/course/240x135/396876_cc92_7.jpg",
-        "title": "Learning Python for Data Analysis and Visualization",
-        "instructor": "Jose Portilla",
-        "rating": ["4.4", "17,956"],
-        "sale": true,
-        "original-price": "1599.99",
-        "new-price": "269.99",
-        "best-seller": true
-
-    },
-    {
-        "id": 3,
-        "image": "https://img-c.udemycdn.com/course/480x270/405878_e5a0_3.jpg",
-        "title": "Python for Beginners - Learn Programming from scratch",
-        "instructor": "Edwin Diaz, Coding Faculty Solutions",
-        "rating": ["4.4", "1,734"],
-        "sale": true,
-        "original-price": "679.99",
-        "new-price": "199.99",
-        "best-seller": false
-    },
-    {
-        "id": 4,
-        "image": "https://img-c.udemycdn.com/course/480x270/426570_1b91_3.jpg",
-        "title": "Learn Python: Python for Beginners",
-        "instructor": "Abrar Hussain",
-        "rating": ["4.2", "2,759"],
-        "sale": true,
-        "original-price": "319.99",
-        "new-price": "199.99",
-        "best-seller": true
-    },
-    {
-        "id": 5,
-        "image": "https://img-c.udemycdn.com/course/480x270/449532_2aa9_7.jpg",
-        "title": "Python Beyond the Basics - Object-Oriented Programming",
-        "instructor": "Infinte Skills",
-        "rating": ["4.4", "2,927"],
-        "sale": true,
-        "original-price": "519.99",
-        "new-price": "199.99",
-        "best-seller": false
+import Tab from './Tab.js'
+import TabContent from './TabContent.js'
+
+class Courses extends Component {
+
+    constructor(props) {
+        super(props)
+        let homePageData;
+        this.state = {
+            isLoading: true,
+            post: {},
+            error: "",
+            mostReleventTab: 0
+
+        }
+    }
+
+    componentDidMount() {
+        this.setState({
+            isLoading: true,
+            error: "",
+            post: {}
+        })
+        fetch('https://ahmadt198.github.io/Data/db.json')
```

Comment:
you can store the url in a file that will be make the change in the future more easier

Reply:
Done